apps/bfd-db-migrator/pom.xml

```diff
@@ -134,6 +132,7 @@
 			<!-- JDBC driver for working with PostgreSQL DBs on Java 8+ (JDBC 4.2). -->
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
+			<version>${postgres.driver.version}</version>
```

Comment:
looks like postgres is in dependency management so we don't need this (it doesn't hurt anything but would be best to take it out). I didn't realize it was already in dep mgmt yesterday.